README.md

```diff
@@ -204,6 +204,17 @@ export PINECONE_ENV="Your pinecone region" # something like: us-east4-gcp
 
 ```
 
+## Plugins
+
+See https://github.com/Torantulino/Auto-GPT-Plugin-Template for the template of the plugins.
+
+WARNING: Review the code of any plugin you use.
```

Comment:
I think this needs to be further stressed.
As you've said, there's plenty of room for malicious plugins.

Reply:
✅